### ISSUE TITLE
@celo/contractkit does not install on Windows

### ISSUE THREAD

**Issue by silentlight** [celo-org/celo-monorepo#6883]
### Expected Behavior
Install @celo/contractkit without errors on Windows

### Current Behavior
When I run npm install --save @celo/contractkit I receive error:

```
npm ERR! code 128
npm ERR! command failed
npm ERR! command git ls-remote ssh://git@github.com/umpirsky/country-list.git
npm ERR! Warning: Permanently added the RSA host key for IP address '13.234.210.38' to the list of known hosts.
npm ERR! git@github.com: Permission denied (publickey).
npm ERR! fatal: Could not read from remote repository.
npm ERR!
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.
```

**Comment by aslawson** [comment]
Should try to confirm by downloading on windows.

**Comment by aslawson** [comment]
closed duplicate #2842

**Comment by barbaraliau** [comment]
Unable to duplicate. Installed on Windows
- contractkit `v1.2.0`
- node `v10.16.0`

I did originally run into a weird `node-gyp` error, but I think it's a bizarre race condition with how the modules are installed. when I ran `npm i` again I was able to successfully install contractkit.
